snow/consensus/snowball/binary_snowflake.go

```diff
@@ -49,6 +49,15 @@ func (sf *binarySnowflake) RecordSuccessfulPoll(choice int) {
 	sf.binarySlush.RecordSuccessfulPoll(choice)
 }
 
+func (sf *binarySnowflake) RecordPollPreference(choice int) {
+	if sf.finalized {
+		return // This instance is already decided.
+	}
+
+	sf.confidence = 0
```

Comment:
nit: can you comment on this? IIUC, a preference reset confidence because it signals there was not a sufficiently large a majority to have a successful poll. Maybe it's a trivial fact, but I'd rather this code errs toward verbosity than the other way around

Reply:
I feel like this is documented by the method description (on the interface)